selfdrive/camerad/cameras/sensor2_i2c.h

```diff
@@ -117,8 +121,10 @@ struct i2c_random_wr_payload init_array_ar0231[] = {
   {0x100C, 0x0589}, // FINE_INTEGRATION_TIME2_MIN
   {0x100E, 0x07B1}, // FINE_INTEGRATION_TIME3_MIN
   {0x1010, 0x0139}, // FINE_INTEGRATION_TIME4_MIN
-  {0x3014, 0x08CB}, // FINE_INTEGRATION_TIME_
```

Comment:
I now have a lot of doubts if this 2 exposure HDR is working at all with these parameters.

Reply:
Maybe I'm just wrong about DIGITAL_CTRL.